src/transformers/__init__.py

```diff
@@ -109,6 +109,7 @@
     CsvPipelineDataFormat,
     FeatureExtractionPipeline,
     FillMaskPipeline,
+    GenerationPipeline,
```

Comment:
Just a naming question, but should it be `TextGenerationPipeline`? The other pipelines are pretty self-explanatorily text-based (Sequence classification, token classification, etc.) but this one isn't. (I don't have a strong opinion on this)

Reply:
I agree

Reply:
Got it. Will work on changing it to `TextGenerationPipeline`!